### PR TITLE
Add LogicalInterface and PhysicalInterface API calls

### PR DESCRIPTION
**Pull request by amitmangalvedkar**
The [IM APIs](https://docs.internetofthings.ibmcloud.com/apis/swagger/v0002/state-mgmt.html) capabilities of IBM® Watson™ IoT Platform help you to organize and integrate data coming in to and going out of Watson IoT Platform. This consists of the following 
1. DeviceTypes
2. Devices
3. EventTypes
4. LogicalInterface
5. PhysicalInterface
6. Schemas

### CODE REVIEW

**Review by CaroMac — CHANGES_REQUESTED** [review]
Suggest rewording 
This sample showcases how to get a device location weather using the Java Client Library
to
The following sample from the Java Client Library retrieves information about the weather at the location of a specified device. 
and
System.out.println("get device location weather of device --> "+DEVICE_ID);
to
System.out.println("get weather information at location of device --> "+DEVICE_ID);

**Review by amitmangalvedkar — COMMENTED** [review]
Accept the changes from forked repos to the main repos

**Review by sathipal — APPROVED** [review]
